_tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api.md

```diff
@@ -49,90 +52,289 @@ GET _remotestore/stats/<index_name>
+* `segment` : Contains stats related to segment transfers from remote-backed storage.
+* `translog` : Contains stats related to translog transfers from remote-backed storage.
+
+#### routing
+
+The `routing` object contains the following fields:
+
+|Field	|Description	|
+|:---	|:---	|
+| `primary` | Denotes if the shard copy is primary shard or not.|
+| `node` | Node name to which the shard is assigned. |
+
+#### segment
+
+The `segment.upload` object contains the following fields:
+
+|Field	|Description	|
+|:---	|:---	|
+| `local_refresh_timestamp_in_millis` | The last successful local refresh timestamp, in milliseconds.  |
+| `remote_refresh_timestamp_in_millis` | The last successful remote refresh timestamp, in milliseconds. |
+| `refresh_time_lag_in_millis` | The time, in milliseconds, the remote refresh is behind the local refresh. |
+| `refresh_lag` | The number of refreshes by which the remote store is lagging behind the local store.   |
+| `bytes_lag` | The lag in bytes between the remote and local store.  |
+| `backpressure_rejection_count` | The total number of write rejections made due to backpressure in the remote store.    |
+| `consecutive_failure_count` | The number of consecutive remote refresh failures since the last successful refresh.  |
+| `total_remote_refresh` | The total number of remote refreshes.  |
+| `total_uploads_in_bytes` | The total number of bytes in all uploads to the remote store.  |
+| `remote_refresh_size_in_bytes.last_successful` | The size of the data uploaded during the last successful refresh.  |
+| `remote_refresh_size_in_bytes.moving_avg` | The average size of data, in bytes, uploaded in the last *N* refreshes. *N* is defined in the `remote_store.moving_average_window_size` setting. For details, see [Remote segment backpressure](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/). |
+| `upload_latency_in_bytes_per_sec.moving_avg` | The average speed of remote segment uploads, in bytes per second, for the last *N* uploads. *N* is defined in the `remote_store.moving_average_window_size` setting. For details, see [Remote segment backpressure](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/).    |
+| `remote_refresh_latency_in_millis.moving_avg` | The average time taken by a single remote refresh during the last *N* remote refreshes. *N* is defined in `remote_store.moving_average_window_size` setting. For details, see [Remote segment backpressure](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/).    |
+
+The `segment.download` object contains the following fields:
+
+|Field	|Description	|
+|:---	|:---	|
+| `last_sync_timestamp`| The timestamp in millisecond since the last successful segement file download from the remote-backed storage. |
```

Comment:
```suggestion
| `last_sync_timestamp`| The timestamp in milliseconds since the last successful segment file download from the remote-backed storage. |
```